### PR TITLE
Add 1 exclude

### PR DESCRIPTION
**Pull request by swedebugia**

### CODE REVIEW

File: queries/default.sparql [hunk 1]
```diff
@@ -67,6 +66,14 @@ WHERE {
     }.
     ?item wdt:P31 ?not
   }
+  # Remove various comman false postives that are any subclass of
```
Comment (waldyrious) [inline queries/default.sparql]:
```suggestion
  # Remove various common false positives that are any subclass of
```

Comment (waldyrious) [reply queries/default.sparql]: @swedebugia why did you mark this as resolved? The typos are still in the diff of the PR.

Comment (swedebugia) [reply queries/default.sparql]: Sorry, did not know the meaning of the button I clicked.

**Comment by swedebugia** [comment]
I have no idea why it now has conflicts. If someone knows how to fix that I'm all ears.

**Comment by Nudin** [comment]
> I have no idea why it now has conflicts. If someone knows how to fix that I'm all ears.

It conflicts because the typos of your last edits are already corrected in master. You can drop the last commit (`git reset HEAD~`) and force push to github (`git push --force-with-lease`).

**Comment by Nudin** [comment]
The querry times out. Maybe you can fix that. Take a look at lukas' tip in #35 and try to limit recursion-depth.

**Comment by waldyrious** [comment]
> It conflicts because the typos of your last edits are already corrected in master.

That's not exactly the case. The typos fixed in master were done on the [comment](https://github.com/Nudin/makesense/blob/1c5f6b8c964e17d683df575fd47d724f1dd8dc45/queries/default.sparql#L25) for the first `FILTER NOT EXISTS` block; the new `FILTER NOT EXISTS` block introduced in this PR had the same typos (due to copy-paste I assume), and fixing those should not cause conflicts.

The conflict is instead due to this PR touching lines around [those edited in the previous commit](https://github.com/Nudin/makesense/commit/2330ce953a375480b18572dff9313cb7a3a100fb#diff-a0572a7a89ff43c8a54f3d0acdcd9874L36-R37).

I believe to fix this, the conflict needs to be resolved manually. @swedebugia, if you check the checkbox "Allow edits maintainers" on this PR's sidebar, either you or @Nudin should be able to fix the conflict directly in the GitHub interface :)

**Pull request merged**